config/config_test.go

```diff
@@ -9,7 +9,12 @@ import (
 )
 
 func TestDefaults(t *testing.T) {
-	cfg, err := New(newViperWithDefaults())
+	v := viper.New()
+	ConfigViper(v)
+	v.SetDefault("auction_timeouts_ms.max", 250)
```

Comment:
This seems a bit weird...

This test targets the `ConfigViper` method, but this sets extra defaults and then asserts on them. Seems like these should go away, and the assertions replaced with the intended default values.

Reply:
The intended default values for these are the zero values, or a URL that has apparently changed in the past. I thought it was a weak test to test that zero values were still zero values, and I wasn't sure about changing the actual default values in this PR. Could just check for the current URL, and the test failure will prompt us to update the test if the URL changes again.

Reply:
ahh, gotcha. I agree, that makes sense.

In that case let's just remove those assertions. There's no value in testing test code.

Reply:
Already replace, found a couple of ints with non-zero defaults to test for instead.